public/robots.txt

```diff
@@ -1,6 +1,11 @@
 User-agent: *
 
-Disallow: /patterns
+Disallow: /checkout
+Disallow: /healthcheck
 Disallow: /manage
+Disallow: /management/manifest
+Disallow: /p/*
```

Comment:
I don't _think_ it makes sense to index `/p/*` pages, as they're generally short-lived if they're only used for time-limited promotions. Unless anyone disagrees?

Reply:
👍 but never-say-never :)